zrml/prediction-markets/src/lib.rs

```diff
@@ -1379,6 +1392,8 @@ mod pallet {
                 T::Swap::set_pool_as_stale(&market.market_type, pool_id, outcome_report)?;
             }
 
+            T::LiquidityMining::distribute_market_incentives(market_id)?;
```

Comment:
I see that this function is called in courts resolve and prediction markets resolve functions. Is there any way that this function could be called twice for one market?

Reply:
`T::LiquidityMining::distribute_market_incentives(market_id)?;` must be called once for every market resolution and this constraint will be enforced when #167 is merged